src/dataexplorer/dataexplorer-application/src/components/Navigation/Navigation.tsx

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import { Menu, Layout} from 'antd';
+import 'antd/dist/antd.css';
+import Dashboard from '../Dashboard/Dashboard'
+import MineSkjema from '../MineSkjema/MineSkjema'
+import MinePasienter from '../MinePasienter/MinePasienter'
+import {Switch, Route, Link, withRouter } from 'react-router-dom'
+import MenuItem from 'antd/lib/menu/MenuItem';
+
+const { Header, Content, Sider, Footer} = Layout;
+const data =
+  {
+  "items": [
+    {
+      "id": 1,
+      "url": "/",
+      "title": "Hjem"
+    },
+    {
+      "id": 2,
+      "url": "/Skjema",
+      "title": "Mine Skjema"
+    },
+    {
+      "id": 3,  
+      "url": "/Pasient",
+      "title": "Mine pasienter"
+    },
+    {
+      "id": 4,
+      "url": "/Search",
+      "title": "Filtrer på egenskaper"
+    },
+  ]
+  }
+
+// for (let index = 0; index < array.length; index++) {
```

Comment:
Don't think the commented code is needed for later use. Maybe remove these comments?

Reply:
Agreed